hub/core/tensor.py

```diff
@@ -104,11 +104,7 @@ def __init__(
 
         self.chunk_engine = ChunkEngine(self.key, self.storage, self.version_state)
         self.index.validate(self.num_samples)
-        self.info = load_info(
```

Comment:
info was getting loaded for every slice of dataset/tensor. This PR addresses that.